vars/githubscm.groovy

```diff
@@ -74,6 +74,27 @@ def mergeSourceIntoTarget(String repository, String sourceAuthor, String sourceB
     """
 }
 
+def forkRepo(String credentialID='kie-ci') {
+    withCredentials([usernamePassword(credentialsId: credentialID, usernameVariable: 'GITHUB_USER', passwordVariable: 'GITHUB_PASSWORD')]){
+        sh 'git config --global hub.protocol https'
+        sh "hub fork --remote-name=origin"
+        sh 'git remote -v'
+    }
+}
+
+def createPR(String targetBranch, String pullRequestMessage, String credentialID='kie-ci') {
+    withCredentials([usernamePassword(credentialsId: credentialID, usernameVariable: 'GITHUB_USER', passwordVariable: 'GITHUB_PASSWORD')]){
+        try{
+            def pullRequestLink = sh(returnStdout: true, script: "hub pull-request -m '${pullRequestMessage}' -b '${targetBranch}' ")
```

Comment:
please add a `.trim()` to the output of the shell command

Reply:
Done